Settings.coffee

```diff
@@ -31,6 +37,8 @@ else
 
 for file in possibleConfigFiles
 	if fs.existsSync(file)
+		if file.endsWith('.coffee')
+			console.warn "CoffeeScript settings file #{file}.coffee is deprecated, please convert to JavaScript"
```

Comment:
```suggestion
			console.warn "CoffeeScript settings file #{file} is deprecated, please convert to JavaScript"
```

If `file` can `.endsWith('.coffee')`, then this shouldn't be needed, right?

Reply:
You're right, thanks 👍🏼